woocommerce/wc-admin
# Add hide task list endpoint and data actions

Adds data store actions for hiding task lists and a REST endpoint.

### Detailed test instructions:

1. Make a `POST` request to `/wp-json/wc-admin/onboarding/tasks/setup/hide`
2. Make sure the task list is hidden (or check the option `woocommerce_task_list_hidden_lists`)
3. Delete the `woocommerce_task_list_hidden_lists` option
4. In your browser console, enter `window.wp.data.dispatch('wc/admin/onboarding').hideTaskList('setup');`
5. Make sure it works to hide the setup task list option (it will not yet be rendered in the front-end until refresh)
